arangod/Agency/Agent.cpp

```diff
@@ -919,22 +919,22 @@ void Agent::lastAckedAgo(Builder& ret) const {
   if (leading()) {
     ret.add(VPackValue("lastAcked"));
     VPackObjectBuilder b(&ret);
-    for (auto const& [key, pair] : lastAckedIndex) {
-      auto lsit = lastSent.find(key);
+    for (auto const& keyVal : lastAckedIndex) {
+      auto list = lastSent.find(keyVal.first);
```

Comment:
The result of `find` is an iterator and not a list.
It looks like `lsit` was intentional here... ls = last, it = iterator, lsit = last iterator...